src/dune_rules/lib_rules.ml

```diff
@@ -457,10 +458,19 @@ let cctx (lib : Library.t) ~sctx ~source_modules ~dir ~expander ~scope
   let js_of_ocaml =
     Js_of_ocaml.In_context.make ~dir lib.buildable.js_of_ocaml
   in
+  (* XXX(anmonteiro): `public_lib_name` is used to derive Melange's
```

Comment:
Just curious: what does XXX stand for, or what is the meaning? I generally thought it's used on temporary changes (to replace FIXME, TODO and such). But in this case, the change is most probably permanent right?

Reply:
I generally use it in the way this answer explains: https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

Perhaps a `NOTE:` would be more appropriate in this case.